Makefile

```diff
@@ -408,10 +408,6 @@ bazel_coverage_test: failpoint-enable bazel_ci_prepare
 		--build_event_json_file=bazel_1.json --@io_bazel_rules_go//go/config:cover_format=go_cover \
 		-- //... -//cmd/... -//tests/graceshutdown/... \
 		-//tests/globalkilltest/... -//tests/readonlytest/... -//br/pkg/task:task_test -//tests/realtikvtest/...
-	bazel $(BAZEL_GLOBAL_CONFIG) coverage $(BAZEL_CMD_CONFIG) \
```

Comment:
featuretag will also used by other feature. Do not delete it.

Reply:
We should not enable it only for distributing reorg, so I remove it now.

Reply:
Do u mean that we can add it back when it is used by other features?